config/install/field.field.media.video.field_video.yml

```diff
@@ -11,11 +11,14 @@ field_name: field_video
 entity_type: media
 bundle: video
 label: 'Video URL'
-description: ''
+description: 'Enter a YouTube/Vimeo video URL. '
```

Comment:
A lot more providers are supported. Not sure how to solve that. Maybe check if there is any list on video_embed_field module and link to it?